internal/querynode/flow_graph_filter_dm_node.go

```diff
@@ -118,7 +118,7 @@ func (fdmNode *filterDmNode) Operate(in []flowgraph.Msg) []flowgraph.Msg {
 				metrics.QueryNodeConsumeCounter.WithLabelValues(strconv.FormatInt(paramtable.GetNodeID(), 10), metrics.DeleteLabel).Add(float64(proto.Size(&resMsg.DeleteRequest)))
 			}
 		default:
-			log.Warn("invalid message type in filterDmNode",
+			log.Info("invalid message type in filterDmNode",
```

Comment:
why is different with `in == nil`

Reply:
It is certain that there are some messages (such as ddl messages) in the dmChannel that the query does not process.
So it seems weird to keep warning?
Under normal circumstances, nil will not be passed in